src/site/_filters/contains-tag.js

```diff
@@ -17,7 +17,7 @@
 // Check to see if the post contains a specific tag.
```

Comment:
Since we're editing this file, what do you think about changing this comment to a `@fileoverview`? I want to see if we can start enforcing that via linting so may as well start now :D

Reply:
I removed the contains-tag filter as it was mostly u used by now.